### PR TITLE
feat : added authentication route

### PR DESCRIPTION
**Pull request by shaikhhamza11**
Screenshot (41)
Screenshot (42)
Screenshot (40)

### CODE REVIEW

**Comment by HrishikeshMahalle** [comment]
Hey hamza the Page looks amazing

**Comment by Roshannair24** [comment]
Looks good to merge

**Review by Roshannair24 — APPROVED** [review]

File: css/login.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+.button {
+  padding: 1.5rem;
+  color: var(--bg-primary);
+  border-radius: 1rem;
+  min-width: 10rem;
+  cursor: pointer;
+}
+.forget-password {
+  color: rgb(15, 15, 214);
+}
```
Comment (HrishikeshMahalle) [inline css/login.css:46]: liked the way u've used rgb.. but variables are always preferred, but any waz what an amazing job man!! carry on...

**Review by HrishikeshMahalle — APPROVED** [review]

File: css/login.css (hunk 1 above)
Comment (shaikhhamza11) [reply css/login.css:46]: will add in next pr

File: css/login.css [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+.button {
+  padding: 1.5rem;
+  color: var(--bg-primary);
+  border-radius: 1rem;
+  min-width: 10rem;
+  cursor: pointer;
+}
+.forget-password {
+  color: rgb(15, 15, 214);
```
Comment (sanjitsarkar) [inline css/login.css:45]: use variable for colors

**Pull request merged**